### ISSUE TITLE
How to configure proxy rules?

### ISSUE THREAD

**Issue by darellwu** [mohithreddy461999/Proxifier-For-Linux#8]
It is easy to find the configuration proxy rules on the windows interface, linux proxifier.conf, how to configure it? Don't know how to write this at all?

The exported configuration file of windows is as follows:

`<RuleList>
<Rule enabled="true">
<Name>...</Name>
<Targets>...</Targets>
<Action type="Proxy">...</Action>
</Rule>
</RuleList>
`

**Comment by m0hithreddy** [comment]
I am not sure of what windows application's configuration file you are talking about. But talking of where you can find the config for this project, it is copied to /usr/local/etc/proxifier.conf (Web reference: https://github.com/m0hithreddy/Proxifier-For-Linux/blob/main/proxifier.conf). I would say you need to change proxy_server_address, proxy_server_port, proxy_server_username, proxy_server_password. The rest leave them to be. If you have any doubts on what option does what let me know. Thanks.

PS: I created this project at a point in my IT journey. There are better config file formats out there, I know. I would love to take some time to further refine this project. Let's see!

**Comment by darellwu** [comment]
The configuration of the file /usr/local/etc/proxifier.conf feels too little, not enough, I can’t attach a picture of my interface for configuring proxy rules on windows .

It is recommended that the author try the Profixer of windows and Cooperate and try Shadowsocks.

**Comment by m0hithreddy** [comment]
Oh I got what you are saying now . Yeah you might find some config rules like exclude applications or some extra options in the windows interface you are talking about. Currently whatever options you are seeing in proxifier.conf are only the supported ones and those are enough to make bare minimum proxy tunneling work. Do you see some important option being left out?

**Comment by darellwu** [comment]
yes，There are no options on the window. I don’t know if you can see the pictures I posted, you can look at it.
[https://images.gitee.com/uploads/images/2021/1124/180156_c519590f_7525182.jpeg](url)

You can paste that url address to the chrome browser address to see.

**Comment by m0hithreddy** [comment]
Yep I can see but I am not familiar with the language. But I get the point. The current application state is not as feature rich as the one you are trying to share.

**Comment by darellwu** [comment]
I am so sorry to here that. Come on, close the loop.
